spec/system/creating_a_journal_spec.rb

```diff
@@ -127,10 +127,12 @@
         expect(page).to have_content("We are in the year-end closing window.")
         click_button "Create Journal"
         wait_for_ajax
+        # Sometimes the first click doesn't work, so try again
+        click_button "Create Journal" unless page.has_content?("Pending Journal")
```

Comment:
@LeticiaErrandonea what do you think of this idea?  It works for me locally, and I think I like it better than skipping the specs or sinking more time into debugging :) 

If you are not opposed to the approach, then:
1. Apply to the other flaky specs in this file
2. Test if we can remove the changes to rails_helper in this PR